setup.py

```diff
@@ -194,7 +194,7 @@ def build_extension(self, ext: CMakeExtension) -> None:
                     "dfanalyzer": [
                         "bokeh>=2.4.2",
                         "pybind11",
-                        "zindex_py==0.0.1",
+                        "zindex_py@git+https://github.com/hariharan-devarajan/zindex.git",
```

Comment:
This is also to support the new version of zindex from git where I encounterd `-lhwloc` issue on Polaris. let me know whether you want to push new version of zindex or just use the git version

Reply:
I just release 0.0.2 for zindex on pypi. Use zindex_py==0.0.2

Reply:
fixed